slides/week6/week6.org

```diff
@@ -0,0 +1,57 @@
+#+TITLE: Week 6
+#+AUTHOR: Sahit Chintalapudi 
+#+EMAIL: schintalapudi@gatech.edu
+
+* Computer Vision
+- We have the knowledge in C++ to describe the logic we might want a robot to
+  have. But we need to be able to make sense of what the robot sees and
+  classify it before we can act on this logic.
+- Cue OpenCV, an open source computer vision library with bindings for C++
+  (and a few other languages)
+- I guess our ability to see has been ++'d
+
+* What does an Image look like to your computer?
+- OpenCV stores images in an object called a /Mat/
+- A Mat is matrix with rows and columns! In a grayscale image. each element
```

Comment:
Also, `A Mat is matrix with rows and columns`. 

(If I'm understanding `Mat`'s correctly)

If I know what a matrix is: This information is redundant.
If I don't know what a matrix is: This leaves me confused (ie: what is a matrix)

I would suggest just saying an array of rows and columns (we covered arrays before), or just saying it's a table (or similar), or just saying it's a matrix.